.github/workflows/code-quality.yml

```diff
@@ -22,25 +22,27 @@ jobs:
         uses: ./.github/workflows/secrets-setup
         with:
           GITHUB_ACTION_RUN_ENV: development
-          TEMPLE_WALLET_JITSU_TRACKING_HOST: ${{ vars.TEMPLE_WALLET_JITSU_TRACKING_HOST }}
-          TEMPLE_WALLET_JITSU_WRITE_KEY: ${{ vars.TEMPLE_WALLET_JITSU_WRITE_KEY }}
-          TEMPLE_WALLET_EXOLIX_API_KEY: ${{ secrets.TEMPLE_WALLET_EXOLIX_API_KEY }}
-          TEMPLE_WALLET_EVERSTAKE_API_KEY: ${{ secrets.TEMPLE_WALLET_EVERSTAKE_API_KEY }}
-          TEMPLE_WALLET_EVERSTAKE_LINK_ID: ${{ secrets.TEMPLE_WALLET_EVERSTAKE_LINK_ID }}
-          TEMPLE_WALLET_UTORG_SID: ${{ secrets.TEMPLE_WALLET_UTORG_SID }}
-          TEMPLE_WALLET_API_URL: ${{ vars.TEMPLE_WALLET_API_URL }}
-          TEMPLE_WALLET_DEXES_API_URL: ${{ vars.TEMPLE_WALLET_DEXES_API_URL }}
-          TEMPLE_WALLET_MOONPAY_API_KEY: ${{ secrets.TEMPLE_WALLET_MOONPAY_API_KEY }}
-          TEMPLE_WALLET_ROUTE3_AUTH_TOKEN: ${{ vars.TEMPLE_WALLET_ROUTE3_AUTH_TOKEN }}
-          TEMPLE_FIREBASE_CONFIG: ${{ secrets.TEMPLE_FIREBASE_CONFIG }}
-          TEMPLE_FIREBASE_MESSAGING_VAPID_KEY: ${{ secrets.TEMPLE_FIREBASE_MESSAGING_VAPID_KEY }}
-          HYPELAB_API_URL: ${{ vars.HYPELAB_API_URL }}
-          HYPELAB_SMALL_PLACEMENT_SLUG: ${{ vars.HYPELAB_SMALL_PLACEMENT_SLUG }}
-          HYPELAB_HIGH_PLACEMENT_SLUG: ${{ vars.HYPELAB_HIGH_PLACEMENT_SLUG }}
-          HYPELAB_WIDE_PLACEMENT_SLUG: ${{ vars.HYPELAB_WIDE_PLACEMENT_SLUG }}
-          HYPELAB_NATIVE_PLACEMENT_SLUG: ${{ vars.HYPELAB_NATIVE_PLACEMENT_SLUG }}
-          HYPELAB_PROPERTY_SLUG: ${{ vars.HYPELAB_PROPERTY_SLUG }}
-          HYPELAB_ADS_WINDOW_URL: ${{ vars.HYPELAB_ADS_WINDOW_URL }}
+          TEMPLE_WALLET_JITSU_TRACKING_HOST: mock
+          TEMPLE_WALLET_JITSU_WRITE_KEY: mock
+          TEMPLE_WALLET_EXOLIX_API_KEY: mock
+          TEMPLE_WALLET_EVERSTAKE_API_KEY: mock
+          TEMPLE_WALLET_EVERSTAKE_LINK_ID: mock
+          TEMPLE_WALLET_UTORG_SID: mock
+          TEMPLE_WALLET_API_URL: mock
+          TEMPLE_WALLET_DEXES_API_URL: mock
+          TEMPLE_WALLET_MOONPAY_API_KEY: mock
+          TEMPLE_WALLET_ROUTE3_AUTH_TOKEN: mock
+          TEMPLE_FIREBASE_CONFIG: mock
+          TEMPLE_FIREBASE_MESSAGING_VAPID_KEY: mock
+          HYPELAB_API_URL: mock
+          HYPELAB_SMALL_PLACEMENT_SLUG: mock
+          HYPELAB_HIGH_PLACEMENT_SLUG: mock
+          HYPELAB_WIDE_PLACEMENT_SLUG: mock
+          HYPELAB_NATIVE_PLACEMENT_SLUG: mock
+          HYPELAB_PROPERTY_SLUG: mock
+          HYPELAB_ADS_WINDOW_URL: mock
```

Comment:
I wouldn't put mock values here because the previous version of the code quality job reminds about missing env variables